scripting/shavit-timelimit.sp

```diff
@@ -35,18 +36,27 @@
 Database gH_SQL = null;
 
 // base cvars
+ConVar mp_do_warmup_period = null;
+ConVar mp_freezetime = null;
+ConVar mp_ignore_round_win_conditions = null;
 ConVar gCV_TimeLimit = null;
 ConVar gCV_RoundTime = null;
 ConVar gCV_RestartGame = null;
```

Comment:
TimeLimit/RoundTime/RestartGame are also variables that exist in the game, therefore should have their proper `sv_` or `mp_` prefix.